### ISSUE TITLE
broken link for "Available Allocation Opportunities" in ACCESS Allocation Policies v1 doc

### ISSUE THREAD

**Issue by mollycule** [access-ci-org/Integration_Roadmaps#86]
Both "Available Allocation Opportunities" bullets in [ACCESS Allocation Policies v1|https://readthedocs.access-ci.org/projects/integration-roadmaps/en/latest/tasks/ACCESS_Allocation_Policies_v1.html] link to [an empty page|https://allocations.access-ci.org/prepare-requests-overview].

This will likely require reaching out to the document's Task Expert directly (via Slack?) or sending a [ticket to ACCESS Allocations|https://support.access-ci.org/open-a-ticket] to update the documentation appropriately.

**Comment by mollycule** [comment]
Oh, the desired link may end up being https://allocations.access-ci.org/project-types

**Comment by jpnavarro** [comment]
Yes, you can reach out to the Task SME through Slack or by opening an Integration and Operation Support Request at https://access-ci.atlassian.net/servicedesk/customer/portal/2/group/3/create/32 using issue type "ACCESS-wide: Provider Integration - Infrastructure Integration and Roadmap", and assign it to the SME.

I would reach out to them even if you think you found the correct link because it reminds them that they need to update their tasks when URLs change on their website.

**Comment by mollycule** [comment]
Yes, I agree that'll be a good path forward. I'm only creating this github issue to document the problem and make everyone aware, but I'm focused on building out the pilot roadmap proposal right now which is why I didn't assign the issue to myself... for now, at least :)
